src/fsharp/Microsoft.FSharp.Compiler/Microsoft.FSharp.Compiler.csproj

```diff
@@ -17,7 +17,7 @@
       <AdditionalProperties>TargetFramework=netcoreapp3.1</AdditionalProperties>
     </ProjectReference>
     <ProjectReference Include="..\FSharp.Build\FSharp.Build.fsproj">
-      <AdditionalProperties>TargetFramework=netcoreapp3.1</AdditionalProperties>
+      <AdditionalProperties>TargetFramework=netstandard2.0</AdditionalProperties>
```

Comment:
If the TFM of FSharp.Build is only netstandard2.0, this shouldn't be necessary.

Reply:
Yeah you would think .... however ... the proto project is a bit wierder than it looks, like above we can get together and you can experience the wierdness if you like.